### PR TITLE
Refactor and clean up e2e/framework/exec_util.go

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Refactor and clean up e2e/framework/exec_util.go

**Which issue(s) this PR fixes**:

Ref #76206 (parent issue #75601 )

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/framework/exec_util.go [hunk 1]
```diff
@@ -131,7 +128,7 @@ func (f *Framework) ExecShellInPod(podName string, cmd string) string {
 	return f.execCommandInPod(podName, "/bin/sh", "-c", cmd)
 }
 
-// ExecShellInPodWithFullOutput executes the specified command on the pod with returing return stdout, stderr and error.
+// ExecShellInPodWithFullOutput executes the specified command on the pod with returning stdout, stderr and error.
```
Comment (neolit123) [inline test/e2e/framework/exec_util.go]: > ExecShellInPodWithFullOutput executes the specified command on the Pod and returns stdout, stderr and error.

**Review by neolit123 — COMMENTED** [review]
thanks @SataQiu 
added one minor comment.

/approve

**Review by fejta — APPROVED** [review]
/lgtm
/approve
thanks!

**Pull request merged**